src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -1203,6 +1203,8 @@ void runtime::cpu::CPU_ExternalFunction::register_common_passes(
     };
 
     REGISTER_KNOBBED_PASS(LikeReplacement, true, ngraph::pass)
+    REGISTER_KNOBBED_PASS_WITH_ARGS(
+        CoreFusion, true, ngraph::pass, ngraph::pass::FusionType::ALL_FUSIONS)
```

Comment:
Instead of moving CoreFusion up, add a second FusedOpDecomposition after CoreFusion.